civis/tests/test_resources.py

```diff
@@ -245,3 +245,38 @@ def test_create_method_unexpected_kwargs():
     with pytest.raises(TypeError) as excinfo:
         method(mock_endpoint, foo=0, bar=0, baz=0)
     assert str(excinfo.value) == expected_msg
+
+
+@mock.patch('builtins.open', new_callable=mock.mock_open,
+            read_data='{"test": true}')
+@mock.patch('civis.resources._resources.generate_classes')
+@mock.patch('civis.resources._resources.parse_swagger')
```

Comment:
For these two lines, you should also pass `autospec=True`. https://docs.python.org/3/library/unittest.mock.html#autospeccing